City-of-Helsinki/drupal-helfi-platform-config
# UHF-4651 install editorially

# Install editorially [UHF-4651](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4651)
Editoria11y module allows content creator to see accessibility violations while browsing through content.

## What was done
* Installed & enabled & configured editoria11y module

## How to install
* Test on any site with existing content. Module will be installed on all sites.
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-4651_install_editorially`
* Run `make drush-updb drush-cr`

## How to test
* Go and view any content page or TPR entity page.
* On bottom right corner should be indicator which shows accessibility violations (If not, then the content is accessible, select another page until you see the indicator)
